repo/packages/M/mongodb-admin/0/marathon.json.mustache

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "{{service.name}}",
+    "cpus": {{mongodb-admin.cpus}},
+    "mem": {{mongodb-admin.mem}},
+    "instances": 1,
+    "env": {
+        {{#auth.enable}}
+        "PASSWORD": "{{auth.password}}",
+        {{/auth.enable}}
+        "CONN_NAME": "{{database.db_location}}.marathon.l4lb.thisdcos.directory:{{database.db_port}}",
+        "DB_USERNAME": "{{database.db_username}}",
+        "DB_PASSWORD": "{{database.db_password}}", 
+        "DB_HOST": "{{database.db_location}}.marathon.l4lb.thisdcos.directory",
+        "DB_PORT": "{{database.db_port}}"             
+    },
+    "container": {
+        "type": "DOCKER",
+        "volumes": [{
+            "containerPath": "var",
+            "hostPath": "{{storage.host_volume}}/{{service.name}}",
+            "mode": "RW"
+        }],
+        "docker": {
+            "image": "{{resource.assets.container.docker.mongodb-admin-docker}}",
+            "forcePullImage": false,
```

Comment:
forcePullImage:false is included by default by Marathon, so I left it there. Thanks for the tip, I'll remove it in further versions.